docs/BreakingChanges/086 Scrolling a WPF TreeView or grouped ListBox in a VirtualizingStackPanel can cause a hang.md

```diff
@@ -0,0 +1,25 @@
+## 86: Scrolling a WPF TreeView or grouped ListBox in a VirtualizingStackPanel can cause a hang
```

Comment:
{
  "Id": "86",
  "Title": "Scrolling a WPF TreeView or grouped ListBox in a VirtualizingStackPanel can cause a hang",
  "Details": "In the .NET Framework v4.5, scrolling a WPF TreeView or grouped ListBox in a virtualized stack panel can caused hangs if there are margins in the viewport (between the items in the TreeView or ListBox, for example, or on an ItemsPresenter element). Additionally, in some cases, different sized items in the view can cause instability even if there are no margins.",
  "Suggestion": "This bug can be avoided by upgrading to .NET Framework 4.5.1. Alternatively, margins can be removed from view collections (like TreeViews) within virtualized stack panels if all contained items are the same size.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/763639/wpf-application-built-on-net-4-0-freezes-under-system-with-net-4-5-while-scrolling-the-treeview-under-specific-conditions",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Windows.Controls.VirtualizingStackPanel.SetIsVirtualizing(System.Windows.DependencyObject,System.Boolean)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5.1",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 1
}

**Parsed APIs:**
- [x] M:System.Windows.Controls.VirtualizingStackPanel.SetIsVirtualizing(System.Windows.DependencyObject,System.Boolean)